app/views/users/new.html.erb

```diff
@@ -1,3 +1,24 @@
 <% provide(:title, 'Sign up') %>
 <h1>Sign up</h1>
-<p>This will be a signup page for new users.</p>
\ No newline at end of file
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
```

Comment:
@shaniajubelag 
This form is not being centered properly : (screenshot) https://prnt.sc/og3hab

Instead of using the **offset** class, you can use this bootstrap class `m-auto` to make it centered.

Reply:
I used bootstrap class `justify-content-center`